### PR TITLE
use the same version of kubectl with kubelet

### PR DESCRIPTION
**Pull request by hchenxa**
**What this PR does / why we need it**:

the `master-user-data.sh` did not specify the version of `kubectl`, it's better to make `kubectl`, `kubelet` and `kubeadm` use the same version.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

### CODE REVIEW

**Comment by hchenxa** [comment]
cc @gyliu513

File: cmd/clusterctl/examples/openstack/provider-component/user-data/ubuntu/templates/master-user-data.sh [hunk 1]
```diff
@@ -94,9 +94,12 @@ function getversion() {
 }
 KUBELET=$(getversion kubelet ${KUBELET_VERSION}-)
 KUBEADM=$(getversion kubeadm ${KUBELET_VERSION}-)
+KUBECTL=$(getversion kubectl ${KUBELET_VERSION}-)
 apt-get install -y \
     kubelet=${KUBELET} \
-    kubeadm=${KUBEADM}
+    kubeadm=${KUBEADM} \
```
Comment (gyliu513) [inline cmd/clusterctl/examples/openstack/provider-component/user-data/ubuntu/templates/master-user-data.sh]: @hchenxa Seems the `kubeadm init` can help to install `kubectl` as well, can you help check if the `kubeadm init` will overwrite the `kubectl` installed by `apt-get install`?

Comment (gyliu513) [reply cmd/clusterctl/examples/openstack/provider-component/user-data/ubuntu/templates/master-user-data.sh]: CentOS has same logic as https://github.com/kubernetes-sigs/cluster-api-provider-openstack/blob/master/cmd/clusterctl/examples/openstack/provider-component/user-data/centos/templates/master-user-data.sh#L62 , I think it is OK to have this PR.

**Comment by hchenxa** [comment]
and here is the output without this PR:
```
root@hchenxa1-master-qkcvs:~# kc version
Client Version: version.Info{Major:"1", Minor:"14", GitVersion:"v1.14.1", GitCommit:"b7394102d6ef778017f2ca4046abbaa23b88c290", GitTreeState:"clean", BuildDate:"2019-04-08T17:11:31Z", GoVersion:"go1.12.1", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"14", GitVersion:"v1.14.0", GitCommit:"641856db18352033a0d96dbc99153fa3b27298e5", GitTreeState:"clean", BuildDate:"2019-03-25T15:45:25Z", GoVersion:"go1.12.1", Compiler:"gc", Platform:"linux/amd64"}
```

And after apply the PR:
```
root@hchenxa1-master-lc79s:~# kc version
Client Version: version.Info{Major:"1", Minor:"14", GitVersion:"v1.14.0", GitCommit:"641856db18352033a0d96dbc99153fa3b27298e5", GitTreeState:"clean", BuildDate:"2019-03-25T15:53:57Z", GoVersion:"go1.12.1", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"14", GitVersion:"v1.14.0", GitCommit:"641856db18352033a0d96dbc99153fa3b27298e5", GitTreeState:"clean", BuildDate:"2019-03-25T15:45:25Z", GoVersion:"go1.12.1", Compiler:"gc", Platform:"linux/amd64"}
```

**Pull request merged**